telamon-cli/src/lib.rs

```
@@ -357,6 +354,90 @@ mod cuda_reference {
 #[cfg(feature = "cuda")]
 pub use cuda_reference::CublasHandle;
 
+#[cfg(feature = "x86")]
+mod x86_reference {
+    use telamon_kernels::linalg;
+
+    use super::Reference;
+
+    #[derive(Default)]
+    pub struct X86Reference {
+        _priv: (),
+    }
+
+    impl<'a> Reference<'a, linalg::Axpy<'a, f32>> for X86Reference {
+        type Context = telamon_x86::Context;
+
+        fn eval_reference(&self, _params: &(i32, bool), _context: &Self::Context) -> f64 {
+            // TODO
+            1.
```

Rather `panic!` instead of returning an arbitrary value